tests/fe/br_approximation_01.cc

```diff
@@ -0,0 +1,518 @@
+// ---------------------------------------------------------------------
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+// ---------------------------------------------------------------------
+
+
+// projects the shape functions from the Bernardi-Raugel elements
+// into different polynomial spaces and computes the error of the
+// approximation. The error should be close to machine precision
+// for polynomials of degree 2 or higher
+
+// this is almost a verbatim copy of rt_approximation_01.cc and
+// may needs additional modifications to make it more robust
+
+#include "../tests.h"
+
+#define PRECISION 8
+
+
+std::ofstream logfile("output");
+
+char buf[1000];
+
+
+#include <deal.II/base/function.h>
+#include <deal.II/base/quadrature_lib.h>
+
+#include <deal.II/dofs/dof_tools.h>
+
+#include <deal.II/fe/fe.h>
+#include <deal.II/fe/fe_abf.h>
+#include <deal.II/fe/fe_bernardi_raugel.h>
+#include <deal.II/fe/fe_dgq.h>
+#include <deal.II/fe/fe_q.h>
+#include <deal.II/fe/fe_raviart_thomas.h>
+#include <deal.II/fe/fe_system.h>
```

Comment:
Please remove the extra finite element header files that we do not need.

Reply:
Got it, thanks!